packages/rules-core/src/enforce-rationale.ts

```diff
@@ -0,0 +1,69 @@
+import { basename } from "path";
+import { PackageRule, FileContext, asBollLineNumber, Result, Success, Failure, Logger } from "@boll/core";
+
+/**
+ * EnforceRationale ensures specified fields in package.json require a rationale for
+ * any additions.
+ */
+const ruleName = "EnforceRationale";
+export class EnforceRationaleRule implements PackageRule {
+  constructor(private logger: Logger, private options?: any) {}
+
+  get name(): string {
+    return ruleName;
+  }
+
+  async check(file: FileContext): Promise<Result[]> {
+    if (basename(file.filename) !== "package.json") {
+      return [new Success(ruleName)];
+    }
+
+    if (!this.options || !this.options.fields) {
+      return [new Success(ruleName)];
+    }
+
+    if (typeof this.options.fields !== "object") {
+      return [
+        new Failure(
+          ruleName,
+          file.filename,
+          asBollLineNumber(0),
+          `The "fields" option for rule ${ruleName} should be an array`
+        )
+      ];
+    }
+
+    const fields = this.options.fields as string[];
+    const contents = JSON.parse(file.content);
+
+    if (fields.length && !contents.rationale) {
```

Comment:
I think this should only happen if `contents` actually declares the field(s) that need rationale but doesn't include the rationale. 

ie, this should be ok:

```json
{
  "name": "foo"
}
```

config:
```js
const options = {
  fields: ['someBadKeyUsage']
```